### PR TITLE
[RA2 Ch3] Memory management

### LINKED ISSUE #1918: [RA2 Ch3] Add memory management content
https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter03.md#3211-memory-management

### PR DESCRIPTION
**Pull request by petorre**
Fixes #1918 .

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter03.md [hunk 1]
```diff
@@ -118,19 +118,19 @@ set of processes that are used to manage these containers (pull, run, delete,
...
-> Relate back to features described in the RM
-[here](../../../ref_model/chapters/chapter05.md#521-virtual-compute). Note that
-the RM appears to be missing Memory-based HW profile features
-[here](../../../ref_model/chapters/chapter05.md#54-nfvi-hw-profiles-features-and-requirements).
+The Reference Model requires the support of Huge Pages in i.cap.018 which is supported by upstream Kubernetes ([documentation](https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/)).
+
+For proper mapping of Huge Pages to scheduled pods, needed is to both have Huge Pages enabled in operating system (configured in kernel and mounted with correct permissions) and kubelet configuration. Multiple sizes of Huge Pages can be enabled like 2 MiB and 1 GiB.
 
 The Reference Model requires the support of Huge Pages in `i.cap.018` which is
 already supported by upstream Kubernetes. For some applications, Huge Pages
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter03.md:129]: Removing a duplication:

```suggestion
For some applications, Huge Pages
```

**Review by tomkivlin — CHANGES_REQUESTED** [review]

File: doc/ref_arch/kubernetes/chapters/chapter03.md [hunk 2]
```diff
@@ -118,19 +118,19 @@ set of processes that are used to manage these containers (pull, run, delete,
...
+<a name="3.2.1.3"></a>
+#### 3.2.1.3 Memory and Huge Pages Resources Management
 
-> Relate back to features described in the RM
-[here](../../../ref_model/chapters/chapter05.md#521-virtual-compute). Note that
-the RM appears to be missing Memory-based HW profile features
-[here](../../../ref_model/chapters/chapter05.md#54-nfvi-hw-profiles-features-and-requirements).
+The Reference Model requires the support of Huge Pages in i.cap.018 which is supported by upstream Kubernetes ([documentation](https://kubernetes.io/docs/tasks/manage-hugepages/scheduling-hugepages/)).
+
+For proper mapping of Huge Pages to scheduled pods, needed is to both have Huge Pages enabled in operating system (configured in kernel and mounted with correct permissions) and kubelet configuration. Multiple sizes of Huge Pages can be enabled like 2 MiB and 1 GiB.
```
Comment (pgoyal01) [inline doc/ref_arch/kubernetes/chapters/chapter03.md:126]: suggest language simplification:
```suggestion
For proper mapping of Huge Pages to scheduled pods, both need to have Huge Pages enabled in the operating system (configured in kernel and mounted with correct permissions) and kubelet configuration. Multiple sizes of Huge Pages can be enabled like 2 MiB and 1 GiB.
```

**Review by pgoyal01 — APPROVED** [review]

**Review by tomkivlin — APPROVED** [review]

**Comment by tomkivlin** [comment]
@petorre could you resolve conflicts please, and then I'll merge.

**Review by rgstori — APPROVED** [review]

**Comment by petorre** [comment]
> 
> 
> @petorre could you resolve conflicts please, and then I'll merge.

@tomkivlin : Done.

**Pull request merged**